test/officerManager.js

```diff
@@ -0,0 +1,336 @@
+        .catch(err => {
+          throw err
+        })
+    })
+
+    it('Should return statusCode 401 if an invalid token was passed in', done => {
+      const form = {
+        token: 'Invalid token',
+        email: 'test@test.com'
+      }
+      chai
+        .request(app)
+        .post('/api/officerManager/edit')
+        .send(form)
+        .then(function (res) {
+          expect(res).to.not.have.status(200)
+
+          done()
+        })
+        .catch(err => {
+          throw err
+        })
+    })
+
+    it('Should return statusCode 404 if no form was found', done => {
+      const form = {
+        token: token,
+        email: 'invalid-email'
+      }
+      chai
+        .request(app)
+        .post('/api/officerManager/edit')
+        .send(form)
+        .then(function (res) {
+          expect(res).to.have.status(404)
+
+          done()
+        })
+        .catch(err => {
+          throw err
+        })
+    })
+
+    it('Should return statusCode 200 and a message if a form was edited', done => {
+      const form = {
+        token: token,
+        email: 'test@test.com',
+        name: 'new name'
+      }
+      chai
+        .request(app)
+        .post('/api/officerManager/edit')
+        .send(form)
+        .then(function (res) {
+          expect(res).to.have.status(200)
+          res.body.should.be.a('object')
+          res.body.should.have.property('message')
+
+          done()
+        })
+        .catch(err => {
+          throw err
+        })
+    })
+  })
+
+  describe('/POST delete', () => {
+    it('Should return statusCode 500 if no token is passed in', done => {
+      const form = {
+        name: 'invalid-name',
+        email: 'test@test.com'
+      }
+      chai
+        .request(app)
+        .post('/api/officerManager/delete')
+        .send(form)
+        .then(function (res) {
+          expect(res).to.not.have.status(200)
+
+          done()
+        })
+        .catch(err => {
+          throw err
+        })
+    })
+
+    it('Should return statusCode 401 if an invalid token was passed in', done => {
+      const form = {
+        token: 'Invalid token',
+        email: 'test@test.com'
+      }
+      chai
+        .request(app)
+        .post('/api/officerManager/delete')
+        .send(form)
+        .then(function (res) {
+          expect(res).to.not.have.status(200)
+
+          done()
+        })
+        .catch(err => {
+          throw err
+        })
+    })
+
+    it('Should return statusCode 404 if no form was found', done => {
+      const form = {
+        token: token,
+        email: 'fsefvsf@dsges.csadw'
+      }
+      chai
+        .request(app)
+        .post('/api/officerManager/delete')
+        .send(form)
+        .then(function (res) {
+          expect(res).to.have.status(404)
+
+          done()
+        })
+        .catch(err => {
+          throw err
+        })
+    })
+
+    it('Should return statusCode 200 and a message if a form was deleted', done => {
```

Comment:
After deleting, please verify that the database should now be empty. This is done after deleting in the Event API, found [here](https://github.com/SCE-Development/Core-v4/blob/7eb49330115efe0619c97377f7b69382cca2d41d/test/Event.js#L284)